3-semantic-html/semantic.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Sematic HTML</title>
+    </head>
+    <body>
+        <header>
+            <h1>Semantic Exercise HTML </h1>
+            <h3>Raúl Valencia</h3>
+            <nav>
+            <ul>
+                <li>Home</li>
+                <li>Profile</li>
+                <li>About of</li>
+                <li>Proyects</li>
+                <li>Contatc</li>
+            </ul> 
+            </nav>
+        </header>
+        <main>
+            <article>
+                <header>
+                    <h1>This is the main</h1>
+                    <blockquote>
+                        "Everybody in this country should learn how to program a computer...
+                        because it teaches you how to think."
+                        <br>
```

Comment:
Using <br> it's a bad practice because it's not a semantic tag. We are going to see how to put a line break with CSS.